src/lib.rs

```diff
@@ -245,16 +265,18 @@ impl DependencyAnalyzer {
 
     /// Try to locate a `lib` that is compatible to `elf`
     fn find_library(&self, elf: &Elf, lib: &str) -> Result<Library, Error> {
-        for ld_path in self
+        for lib_path in self
             .runpaths
             .iter()
             .chain(self.env_ld_paths.iter())
             .chain(self.conf_ld_paths.iter())
+            .map(|ld_path| {
+                self.root
+                    .join(ld_path.strip_prefix('/').unwrap_or(ld_path))
+                    .join(lib)
+            })
+            .chain(self.additional_ld_paths.iter().cloned())
```

Comment:
Fixed in https://github.com/messense/lddtree-rs/commit/f899f3421073bdc194b10d6bd3f10304892cfff7